data/mods/ssb/statuses.ts

```diff
@@ -57,6 +57,22 @@ export const BattleStatuses: {[k: string]: ModdedPureEffectData} = {
 			}
 		},
 	},
+
+	frostyicelad: {
+		noCopy: true,
+		onStart(source) {
+			this.add(`c|${getName('Frostyicelad')}|Oh i guess its my turn now! Time to sweep!`);
+			if (source.species.id !== 'frosmothmega' || source.illusion) return;
+			this.add('-start', source, 'typechange', 'Bug/Ice');
```

Comment:
again not necessary imo

Reply:
until tooltips has it i view it as necessary personally, only reason i added it